tests/testthat/test-param-estimates.R

```diff
@@ -18,14 +18,8 @@ withr::with_options(list(rbabylon.bbi_exe_path = read_bbi_path(),
                          rbabylon.model_directory = NULL), {
 
   test_that("param_estimates.bbi_model_summary gets expected table", {
-
-    # extract parameter df
     par_df <- MOD1 %>% model_summary() %>% param_estimates()
-
-    # for some arcane reason `expect_equal(par_df, ref_df1)` fails, so we just check a few columns
```

Comment:
and this `arcane reason` no longer exists! My guess it that it had something to do with transitioning into tibble 1.0.0 but that's all water under the bridge now. We're back to testing against the golden file explicitly.